### PR TITLE
feat: enable typedoc for new react package

### PR DESCRIPTION
**Pull request by reecejohnson**

### CODE REVIEW

File: packages/react/src/publication/usePublications.ts [hunk 1]
```diff
@@ -15,41 +15,70 @@ import { PaginatedArgs, PaginatedReadResult, usePaginatedReadResult } from '../h
...
+ *      publicationTypes: [PublicationType.Post]
+ *    }
+ * });
+ * ```
+ * @example
+ * Fetch comment publications
+ * ```tsx
+ * const { data, loading, error } = usePublications({
+ *   where: {
+ *      publicationTypes: [PublicationType.Comment]
```
Comment (cesarenaldi) [inline packages/react/src/publication/usePublications.ts:37]: does it work without profile id at all?

Comment (reecejohnson) [reply packages/react/src/publication/usePublications.ts:37]: sorry, just saw this - but yes it does

Comment (reecejohnson) [reply packages/react/src/publication/usePublications.ts:37]: added these examples to show how people could replicate 

`useComments`
`usePosts`
`useMirrors`
`useQuotes`

with this hook

Comment (cesarenaldi) [reply packages/react/src/publication/usePublications.ts:37]: Worth looking at these examples as most likely use case.

If we are to got with `usePublications` for comments (and not implement `useComments`)... which I think it's an interesting idea BTW... maybe a more likely example is "find all comments on a given `publicationId`.

**Pull request merged**